### DISCUSSION TITLE
update

### DISCUSSION THREAD

**Discussion by Electric-Motor** [organicmaps/organicmaps#7817] [Q&A] [answered]
I need to update the map by the github by myself ？

**Comment by hb0nd** [comment, marked as answer]
Yes, there's no in-app updater.

Alternatively, you can use Obtainium (https://github.com/ImranR98/Obtainium), which automatically checks and installs updates from github

**Comment by Electric-Motor** [comment]
Thanks for your reply ！😊

________________________________
